examples/DynaSwing/IEC/Wind/Neplan/WPP4BCurrentSource/WPP4BCurrentSource.dyd

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We should do the same test case for WPA, WPB, WTA, WTB to ease the comparison, thanks !